spyndra:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  base_to_femur_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_femur_1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  base_to_femur_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_femur_2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  base_to_femur_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_femur_3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  base_to_femur_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_femur_4
    pid: {p: 100.0, i: 0.01, d: 10.0}
  femur_to_tibia_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur_to_tibia_1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  femur_to_tibia_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur_to_tibia_2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  femur_to_tibia_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur_to_tibia_3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  femur_to_tibia_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur_to_tibia_4
    pid: {p: 100.0, i: 0.01, d: 10.0}
